context-free.md

```diff
@@ -2,21 +2,45 @@
 
 ##  Who is this programming language for?
 
+Computer-aided drawing of images
```

Comment:
Lots of languages do this, like photoshop or imagemagick. I think you should be more specific. You might also want to answer *who* it is for rather than *what* it is for